src/libs/OptionsListUtils.js

```diff
@@ -204,20 +205,76 @@ function hasReportDraftComment(report) {
         && lodashGet(reportsWithDraft, `${ONYXKEYS.COLLECTION.REPORTS_WITH_DRAFT}${report.reportID}`, false);
 }
 
+/**
+ * @param {Object} report
+ * @returns {Array<String>}
+ */
+function getParticipants(report) {
+    const participants = [...lodashGet(report, ['participants'], [])];
+    if (ReportUtils.isPolicyExpenseChat(report)) {
+        if (!_.contains(participants, report.ownerEmail)) {
+            participants.push(report.ownerEmail);
+        }
+    }
+    return participants;
+}
+
+/**
+ *  Get the Avatar url or fallback to the default icon according to the chat type
+ *
+ * @param {String} source
+ * @param {Object} options
+ * @param {Boolean} [options.isChatRoom]
+ * @param {Boolean} [options.isArchivedRoom]
+ * @param {Boolean} [options.isPolicyExpenseChat]
+ * @returns {String | function():any}
+ */
+function getAvatarSource(source, {isChatRoom, isArchivedRoom, isPolicyExpenseChat}) {
+    if (!source) {
```

Comment:
Why do we always want to return the `source` if we have one? Is there some case where we are specifying the kind of avatar that we want to use and also providing a `source`?

Reply:
Asked another way, can we just return the source when we haven't specified a generic type?

Reply:
I merged these functions into one function that only receives the `report` object, I think it's now cleaner